plugins/toolkit/jetbrains-core/build.gradle.kts

```diff
@@ -144,10 +148,24 @@ tasks.processTestResources {
     // TODO how can we remove this. Fails due to:
     // "customerUploadedEventSchemaMultipleTypes.json.txt is a duplicate but no duplicate handling strategy has been set"
     duplicatesStrategy = DuplicatesStrategy.INCLUDE
+}
 
-    // delete when fully split
-    // pull in shim to make tests pass
-    from(project(":plugin-toolkit:intellij").file("src/main/resources"))
+// delete when fully split
+// pull in shim to make tests pass
```

Comment:
these lines can probably be deleted?

Reply:
this is the comment explaining why it exists